plugin/src/main/java/io/snyk/eclipse/plugin/utils/FileSystemUtil.java

```diff
@@ -19,18 +20,18 @@ public final class FileSystemUtil {
    * @return File
    */
   public static File getCliFile() {
-    return new File(getCliDirectory(), Platform.current().snykWrapperFileName);
+    return new File(new Preferences().getCliPath());
```

Comment:
If I remember correctly, this broke Eclipse tests, because of not allowing injecting a Preference mock.

Reply:
I have rewritten the preferences altogether. Constructor is now private, uses a static factory instead